lib/taurus/qt/qtgui/base/taurusbase.py

```diff
@@ -1342,6 +1361,8 @@ def __init__(self, name='', parent=None, designMode=False, **kwargs):
         self.call__init__(TaurusBaseComponent, name,
                           parent=parent, designMode=designMode)
         self._setText = self._findSetTextMethod()
+        self._formaters = scheme_formatters
+        self._formaters['{:2.3e}'] = '{:2.3e}'
```

Comment:
Instead of defining the exponential formatter here, I would simply register it using the entry point (just as the default formatter)